### PR TITLE
TT-3812 replace react-sortable-hoc witn  hello-pangea/dnd

### PR DESCRIPTION
**Pull request by gtryus**
- updated useMounted to work correctly in development as well as production

- use key to refresh drop target

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -238,5 +238,7 @@
   "engines": {
     "npm": ">=8.0.0 <9.0.0",
     "node": ">=16.0.0 <17.0.0"
-  }
+  },
+  "main": "public/electron.js",
+  "homepage": "./"
```
Comment (sarahentzel) [inline package.json:243]: probably don't want these

File: src/hoc/VertListDnd.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  id: string;
+  content: ReactNode;
+}
+
+// fake data generator
+const getItems = (count: number) =>
+  Array.from({ length: count }, (_v, k) => k).map((k) => ({
+    id: `${k}`,
+    content: `item ${k}`,
+  }));
```
Comment (sarahentzel) [inline src/hoc/VertListDnd.tsx:20]: take this out

File: src/hoc/VertListDnd.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      onDrop({
+        results: newItems.map((item) => item.id),
+        oldIndex: result.source.index,
+        newIndex: result.destination.index,
+      });
+    }
+  };
+
+  // Normally you would want to split things out into separate components.
+  // But in this example everything is just done in one place for simplicity
```
Comment (sarahentzel) [inline src/hoc/VertListDnd.tsx:86]: take this comment out?

**Review by sarahentzel — CHANGES_REQUESTED** [review]

File: src/hoc/VertListDnd.tsx (hunk 2 above)
Comment (gtryus) [reply src/hoc/VertListDnd.tsx:20]: :+1

**Pull request merged**